gitgud/test_operations.py

```diff
@@ -0,0 +1,56 @@
+from git.exc import GitCommandError
+
+from gitgud.skills import all_skills
+from gitgud.operations import get_operator
+
+
+def setup_rebase_conflict():
+    file_operator = get_operator()
+
+    with open("foo", "w") as f:
+        f.write("branch 1")
+    file_operator.repo.index.add("foo")
+    file_operator.repo.index.commit("Add a file")
+
+    file_operator.repo.git.checkout('-b', 'branch', 'HEAD~')
+    with open("foo", "w") as f:
+        f.write("branch 2")
+    file_operator.repo.index.add("foo")
+    file_operator.repo.index.commit("Add a file")
+
+    try:
+        file_operator.repo.git.rebase('master')
+    except GitCommandError:
+        # This will happen every time
+        pass
+
+
+def setup_bisect():
+    file_operator = get_operator()
+    file_operator.repo.git.bisect('start')
+
+
+def test_reset_during_rebase_conflict(gg):
```

Comment:
Minor nitpick, I'd change the testing to use fixtures instead, but that's not the point of this PR. So, feel free to keep it as it is, if you want.

Reply:
We got a pytest